civictechprojects/urls.py

```diff
@@ -19,7 +19,8 @@
 from . import views
 
 urlpatterns = [
-    url(r'^projects/signup/$', views.project_signup, name='project_signup'),
+    url(r'^projects/signup/(?P<project_id>[0-9]+)/$', views.project_edit, name='project_edit'),
```

Comment:
if this is the URL for editing an existing project, seems weird that it has `signup` in the URL, no?

Reply:
Changed